### DISCUSSION TITLE
Invitation to Community Members to Discuss the Development of the Umo Editor

### DISCUSSION THREAD

**Discussion by umodoc** [umodoc/editor#43] [Ideas]
**Dear Community Members,**

We are very grateful for the continuous attention and support you have given to our project. In order to make our project more in line with user needs and better serve everyone, we have initiated a discussion here, hoping to hear voices and suggestions from different perspectives.

### Discussion Purpose

Through this discussion, we hope to achieve the following goals:

1. **Understanding Usage Scenarios**: Collect information on the specific scenarios in which users employ our project, as well as the problems and challenges encountered during the process;
2. **Demand Research**: Explore the specific needs of users for the project's functionality, including the improvement of existing features and expectations for new features;
3. **Suggestions for Improvement**: Invite users to propose concrete suggestions for project improvement, including but not limited to performance optimization, user interface (UI) / user experience (UX) design, and the development of new features;
4. **Development Direction**: Discuss the future development direction of the project, including technological trends, market positioning, and long-term planning, to ensure the project can continue to develop and meet future needs.

#### Usage Scenarios

- How do you use our project?
- Are there any scenarios in the process of using it that particularly require our support?

#### Demand Research

- In what aspects do you think our project needs to be improved?
- Are there any specific features or characteristics that you hope we will add?

#### Suggestions for Improvement

- What specific suggestions do you have for improving our project?
- How can these suggestions help enhance the usability and user experience of the project?

### Discussion Method

We encourage everyone to participate in the discussion with an open and constructive attitude. You can:

- Directly reply with your thoughts and suggestions under this discussion.
- If you have a detailed plan or design, you are welcome to share it in the form of an attachment.
- If you wish to further discuss, you can initiate new topics or subtopics.

### Discussion Rules

- Please maintain respect and friendliness in the discussion, avoiding personal attacks or negative remarks.
- Try to provide **specific, detailed feedback**, which helps us better understand your needs.
- If your ideas gain widespread support from the community, we will consider incorporating them into the project's development plan.

We look forward to communicating with every community member. Each piece of feedback from you is the driving force for our progress. Thank you for your participation and support!

此讨论的中文版本见 #42 ，如果你是中文用户，请移步到 [邀请社区成员共同探讨 Umo Editor 项目的发展](https://github.com/umodoc/editor/discussions/42)。

**Comment by aeterno-caspian** [comment]
I think we should create a community Slack channel or Discord channel to discuss more closely.

**Reply by umodoc** [reply]
I've already created a Discord server: [https://discord.gg/yBwBmm8e](https://discord.gg/yBwBmm8e) — feel free to join!

**Reply by feritarou** [reply]
Hello @umodoc, I'd like to join the Discord server but clicking that link shows an "invalid invitation" page.

**Comment by umodoc** [comment]
Hi! @feritarou Thanks for letting me know — I forgot to set the invitation link to never expire 😅

Here’s the latest invite link:
https://discord.com/invite/k8GjuBBhXD

Looking forward to seeing you there!
